### PR TITLE
[Product Partnerships] Adding quickstart for aws rekognition

### PR DESCRIPTION
**Pull request by aj-NR**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [X] Did you check you NRQL syntax? - Does it work?
- [X] Did you include a Data source and Documentation reference?
- [X] Are all documentation links publicly accessible?
- [X] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [X] Did you check your descriptive content for spelling and grammar errors?
- [X] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [X] Does the PR contain a screenshot for each of your dashboards?
- [X] Do your screenshots show data?
- [X] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [X] Did you check that your alerts actually work?

### CODE REVIEW

File: alert-policies/aws-rekognition/LatencyInResponse.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Name of the alert
+name: Latency In Response
+
+# Description and details
+description: |+
+  The latency for successful requests between the time that the request was made and the response was passed back
```
Comment (seemantk) [inline alert-policies/aws-rekognition/LatencyInResponse.yml:6]:
```suggestion
  This alert occurs when there is a latency in response to successful requests
```

File: quickstarts/aws/aws-rekognition/config.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  ## Monitor your AWS Rekognition and AWS Rekognition Operation Metrics
+
+  With New Relic's Amazon Rekognition integration, you now can see all your Rekognition data on New Relic Instant Observability Platform.
+
+  ### Get started!
+
+   Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-rekognition-monitoring-integration/) to learn more about New
+   Relic monitoring and integration with Amazon Rekognition.
+
+summary: Use New Relic - Amazon Rekognition Integration to analyse your rekognition data with New Relic
```
Comment (seemantk) [inline quickstarts/aws/aws-rekognition/config.yml:12]:
```suggestion
summary: Use New Relic - Amazon Rekognition Integration to analyze your rekognition data with New Relic
```

We standardize on American spelling

**Review by seemantk — COMMENTED** [review]
Great work here Aarushi, just some minor language tweaks that I've suggested.

File: alert-policies/aws-rekognition/HighServerErrors.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Name of the alert
+name: High Server Errors
+
+# Description and details
+description: |+
+  This alert occurs when the number of system errors are more than 10 in 300sec
```
[the remark this answers is not in the document]
Comment (aj-NR) [reply alert-policies/aws-rekognition/HighServerErrors.yml]: addressed

File: alert-policies/aws-rekognition/LatencyInResponse.yml (hunk 1 above)
Comment (aj-NR) [reply alert-policies/aws-rekognition/LatencyInResponse.yml]: addressed

File: quickstarts/aws/aws-rekognition/config.yml (hunk 2 above)
Comment (aj-NR) [reply quickstarts/aws/aws-rekognition/config.yml]: addressed

**Comment by aj-NR** [comment]
> Great work here Aarushi, just some minor language tweaks that I've suggested.

Thanks @seemantk  for the review. I have incorporated the suggestions. Please check.

**Review by mickeyryan42 — APPROVED** [review]

**Pull request merged**